Makefile

```diff
@@ -40,6 +41,9 @@ all: build e2e
 
 build: ignition-server hypershift-operator control-plane-operator konnectivity-socks5-proxy hypershift availability-prober token-minter
 
+build-prod: GO_BUILD_RECIPE+="-ldflags=-s -w"
+build-prod: build
```

Comment:
If we choose to keep debugging symbols (my preference) we don't need a new make target

Reply:
Of course - only called from the Dockerfile